Performance/generators/cpp.sh

```diff
@@ -71,6 +71,7 @@ using google::protobuf::Descriptor;
 using google::protobuf::DescriptorPool;
 using google::protobuf::TextFormat;
 using google::protobuf::util::BinaryToJsonString;
+using google::protobuf::util::JsonParseOptions;
```

Comment:
This seems to indicate that the base branch of the PR isn't correct.  Surely the formatter doesn't add lines?!

Reply:
Also, should we be formatting a `.sh` file?

Reply:
Oh, this is just my fork's `master` being more stale than I realized compared to upstream.

I'd rebase/merge and push again, but I'm afraid GitHub will hide all the existing comment threads once the commit is outdated. There haven't been any terribly significant code changes otherwise here that would affect what we want to look at, though.